validate_email.cwl

```diff
@@ -56,25 +61,27 @@ requirements:
           sub = syn.getSubmission(args.submissionid)
           userid = sub.userId
           evaluation = syn.getEvaluation(sub.evaluationId)
+          message = subject = ""
           if args.status == "INVALID":
             subject = "Submission to '%s' invalid!" % evaluation.name
             message = ["Hello %s,\n\n" % syn.getUserProfile(userid)['userName'],
                        "Your submission (%s) is invalid, below are the invalid reasons:\n\n" % sub.name,
                        args.invalid,
                        "\n\nSincerely,\nChallenge Administrator"]
-          else:
+          elif not args.errors_only:
```

Comment:
Kenny brought this up with me before, and this is just being knit-picky now, but and `if-else` clause should be confined to the same logic.   So:

logically speaking`if args.status == "INVALID", the `elif` or `else` would pertain to the `args.status`.    I understand this `elif` logic though, just for future FYI.

Reply:
Oh interesting.  Okay, I'll keep that in mind for next time!

Reply:
Ya it just makes your code more intuitive to other people that read it. Thanks for the contribution!